demo/index.html

```diff
@@ -58,7 +58,7 @@ <h1><a href="https://github.com/ApoorvSaxena/lozad.js"><strong>Lozad.js</strong>
                 <h1>Lozad.js - Performant Lazy Loading Library</h1>
             </article>
             <article class="thumb">
-                <img class="lozad" data-src="images/thumbs/01.jpg" data-index="1" />
+                <img class="lozad" data-placeholder="#583624" data-src="images/thumbs/01.jpg" data-index="1" />
                 <noscript><img src="images/thumbs/01.jpg" data-index="1" /></noscript>
```

Comment:
@igorkamyshev will be great to have this placeholder color added in all images of demo page 👍